tests/gold_tests/autest-site/ports.py

```diff
@@ -117,7 +115,7 @@ def _get_available_port(queue):
     return port
 
 
-def _setup_port_queue(amount=2000):
+def _setup_port_queue(amount=1000):
```

Comment:
This we will not commit here. This is only for testing that this change helps the test run better.